### ISSUE TITLE
[BUG] 'contents' , 'playerResponse'  of undefined

### ISSUE THREAD

**Issue by MontiKasta** [SuspiciousLookingOwl/youtubei#47]
**Describe the bug**
.getVideo() function returns an undefined error.

**To Reproduce**

**_Note : I'm not writing all the code. I only write the part that needs to be corrected. I used three dots for the remaining parts._**

1. Client.js 
    77 : `if (!response.data[3].response.contents)`
    79 : `return (!response.data[2].playerResponse`...

2. BaseVideo.js 
    58 : `const secondaryContents = data[3].response.contents`...
    100 : `const contents = data[3].response.contents`...
    104 : `const videoDetails = data[2].playerResponse.videoDetails;`
    
3. Video.js     
    39 : `const itemSectionRenderer = data[3].response.contents`...
 
**Additional context**
I think it's give error because something changed inside Youtube or the module used.

**Comment by MontiKasta** [comment]
I got another error in the same file and I think unnecessary to open a new topic. 

TypeError: Cannot read property 'videoPrimaryInfoRenderer' of undefined

**_This is how I did it but I'm not sure if that fixed the problem :_** 

BaseVideo.js 101 - 103

```
const primaryInfo = contents[0].videoPrimaryInfoRenderer;
const secondaryInfo = contents[1].videoSecondaryInfoRenderer;
```

**Comment by SuspiciousLookingOwl** [comment]
Few days ago YouTube changed the response schema (https://github.com/SuspiciousLookingOwl/youtubei/issues/46), seems like they reverted it back 😕

Fixing this soon

**Comment by SuspiciousLookingOwl** [comment]
Should be fixed in `youtubei@0.0.1-rc.35`

**Comment by LuanRT** [comment]
> Should be fixed in `youtubei@0.0.1-rc.35`, expect something breaking again from YouTube soon.. 😅

It seems like the library still uses the old API endpoint with the `pbj` parameter. Sooner or later YouTube will deprecate this.
https://github.com/SuspiciousLookingOwl/youtubei/blob/c19ee3e44d2b881e8ebf594c11020743d524250e/src/classes/Client.ts#L116

You can get the same data by making a request to the `/player` and `/next` endpoints.
